packages/ckeditor5-html-support/tests/integrations/image.js

```diff
@@ -2474,10 +2474,10 @@ describe( 'ImageElementSupport', () => {
 						'alt',
 						'src',
 						'srcset',
-						'linkHref',
 						'width',
 						'height',
 						'placeholder',
+						'linkHref',
```

Comment:
I am unsure about this test. If the order of items in the array does not matter, the assertion should be replaced with `to.contain` (x7). The current implementation it looks like the order should not be changed.